DomainContent/AvatarStore/CheckoutZone/checkoutZone.js

```diff
@@ -0,0 +1,252 @@
+        if (marketplaceID === undefined) {
+            return;
+        }
+        var marketplaceItemTransforms = getTransformForMarketplaceItems();
+        var marketplaceItemTransform = getTransformsForMarketplaceItem(marketplaceID);
+        if (certificateID !== undefined) {
+            marketplaceItemTransform.certificateTransforms[certificateID] = transform;
+        } else {
+            marketplaceItemTransform.unsortedTransforms.push(transform);
+        }
+        marketplaceItemTransforms[marketplaceID] = marketplaceItemTransform;
+        Settings.setValue(TRANSFORMS_SETTINGS, marketplaceItemTransforms);
+    };
+
+    // Get info on checkout stand so we can place copies of items on it for purchasing
+    // Find the position of the top of the stand at one end
+    var getCheckoutStandPosition = (function() {
+        var zoneChildren = Entities.getChildrenIDs(zoneID);
+        zoneChildren.forEach(function (childID) {
+            var name = Entities.getEntityProperties(childID, 'name').name;
+            if (name === "Checkout Table") {
+                tableProperties = Entities.getEntityProperties(childID, ['id', 'position', 'dimensions', 'rotation']);
+                tableID = tableProperties.id;
+                tableHeight = tableProperties.dimensions.y;
+                tableLength = tableProperties.dimensions.x;
+                
+                var halfTableHeight = HALF * tableHeight;
+                var verticalSpace = VERTICAL_SPACING * ITEM_HEIGHT;
+                spawnY = halfTableHeight + verticalSpace;
+                var halfTableLength = HALF * tableLength;
+                spawnZ = (halfTableLength);
+                spawnX = 0;
+            }
+        });
+    });
+
+    // Spawn a copy of each attachment scaled down to fit the ITEM_HEIGHT and place it on the checkout table
+    var spawnOverlayReplica = (function(entityID) {
+        var entityProperties = Entities.getEntityProperties(entityID, [
+            'name', 'modelURL', 'type', 'dimensions', 'marketplaceID', 'modelURL',
+            'localPosition', 'localRotation', 'dimensions', 'parentJointIndex'
+        ]);
+        var overlayProperties = {
+            url: entityProperties.modelURL,
+            name: OVERLAY_PREFIX + entityProperties.marketplaceID,
+            alpha: true,
+            grabbable: true,
+            parentID: tableID,
+            localPosition: {x: spawnX, y: spawnY, z: spawnZ},
+            localRotation: {x: 0, y: 0, z: 0},
+            // clone dimensions so we can alter it without messing up the original entities dimensions
+            dimensions: JSON.parse(JSON.stringify(entityProperties.dimensions))
+        };
+        var scale = (ITEM_HEIGHT / overlayProperties.dimensions.y);
+        if ((overlayProperties.dimensions.x > ITEM_HEIGHT) || (overlayProperties.dimensions.y > ITEM_HEIGHT) || (overlayProperties.dimensions.y > ITEM_HEIGHT)) {
```

Comment:
This line is pretty long, please trim it down to multiple lines of 128 characters max.